src/client/models/Notification.ts

```diff
@@ -0,0 +1,105 @@
+import {Entity} from './Entity';
```

Comment:
Can we remove the `models` directory in the client? Unless the client grows massively, think a flat structure probably makes most sense?